pages/home/script.js

```diff
@@ -482,11 +482,6 @@ const hazardFilterTempApply = async ({ target }) => {
     target.classList.remove('selected');
     // all filters are de-selected
 
-    if (hazardTempFilters.length === 0) {
-      hazardShowCount = 0;
-      showReportsBtnStatus(hazardShowCount);
-      return;
-    }
     hazardShowCount = geoMap.filterMarkerCount(hazardTempFilters);
```

Comment:
This logic was no longer needed as we show `all reports` when `hazardTempFilters` array has no id's in it.